tests/modules/nf-core/ilastik/pixelclassification/nextflow.config

```
@@ -0,0 +1,20 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: "ILASTIK_PIXELCLASSIFICATION_PROBABILITIES" {
+        ext.args = '--output_format=hdf5 --output_filename_format=probabilities.h5 --export_source="Probabilities"'
+    }
+
+        withName: "ILASTIK_PIXELCLASSIFICATION_SIMPLESEGMENTATION" {
+        ext.args = '--output_format=hdf5 --output_filename_format=simple_segmentation.h5 --export_source="Simple Segmentation"'
+    }
+
+        withName: "ILASTIK_PIXELCLASSIFICATION_UNCERTAINTY" {
+        ext.args = '--output_format=hdf5 --output_filename_format=uncertainty.h5 --export_source="Uncertainty"'
+    }
+
+        withName: "ILASTIK_PIXELCLASSIFICATION_FEATURES" {
+        ext.args = '--output_format=hdf5 --output_filename_format=features.h5 --export_source="Features"'
+    }
```

```suggestion
    withName: "ILASTIK_PIXELCLASSIFICATION_PROBABILITIES" {
        ext.args = '--output_format=hdf5 --export_source="Probabilities"'
    }

        withName: "ILASTIK_PIXELCLASSIFICATION_SIMPLESEGMENTATION" {
        ext.args = '--output_format=hdf5 --export_source="Simple Segmentation"'
    }

        withName: "ILASTIK_PIXELCLASSIFICATION_UNCERTAINTY" {
        ext.args = '--output_format=hdf5 --export_source="Uncertainty"'
    }

        withName: "ILASTIK_PIXELCLASSIFICATION_FEATURES" {
        ext.args = '--output_format=hdf5 --export_source="Features"'
    }
```

These should be in the main script (see my next comments)